<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateAccount</class>
 <widget class="QWidget" name="CreateAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>80</y>
     <width>611</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cairo</family>
     <pointsize>20</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Create Account </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>611</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cairo</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>First name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>180</y>
     <width>640</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cairo</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Last name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>180</y>
     <width>611</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cairo</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Phone number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>611</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cairo</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>280</y>
     <width>611</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cairo</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Confirm Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>280</y>
     <width>611</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cairo</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Bio</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createAccount_bt">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>430</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Create account</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="firstName_tb">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="lastName_tb">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>230</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="phonNumber_tb">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>230</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="confirmPassword_tb">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>340</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="password_tb">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="bio_tb">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>340</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
